templates/etc/nginx/sites-available/default.conf.j2

```diff
@@ -797,4 +797,4 @@ server {
 {% endif %}
 }
 
-# vim:filetype=nginx
+# vim:filetype=jinja
```

Comment:
This will mess up the syntax highlighting when generated configuration file is viewed in vim. I guess removing the magic line entirely will be a better choice.